packages/puppeteer-crawler/src/internals/enqueue-links/click-elements.ts

```diff
@@ -49,15 +54,52 @@ export interface EnqueueLinksByClickingElementsOptions {
     clickOptions?: ClickOptions;
 
     /**
-     * An array of {@link PseudoUrl}s matching the URLs to be enqueued,
-     * or an array of strings or RegExps or plain Objects from which the {@link PseudoUrl}s can be constructed.
+     * An array of glob pattern strings or plain objects
+     * containing glob pattern strings matching the URLs to be enqueued.
      *
-     * The plain objects must include at least the `purl` property, which holds the pseudo-URL string or RegExp.
-     * All remaining keys will be used as the `requestTemplate` argument of the {@link PseudoUrl} constructor,
-     * which lets you specify special properties for the enqueued {@link Request} objects.
+     * The plain objects must include at least the `glob` property, which holds the glob pattern string.
+     * All remaining keys will be used as request options for the corresponding enqueued {@link Request} objects.
      *
-     * If `pseudoUrls` is an empty array, `null` or `undefined`, then the function
-     * enqueues all links found on the page.
+     * The matching is always case-insensitive.
+     * If you need case-sensitive matching, use `regexps` property directly.
+     *
+     * If `globs` is an empty array or `undefined`, then the function
+     * enqueues all the intercepted navigation requests produced by the page
+     * after clicking on elements matching the provided CSS selector.
+     */
+    globs?: GlobInput[];
+
+    /**
+     * An array of regular expressions or plain objects
+     * containing regular expressions matching the URLs to be enqueued.
+     *
+     * The plain objects must include at least the `regexp` property, which holds the glob pattern string.
+     * All remaining keys will be used as request options for the corresponding enqueued {@link Request} objects.
+     *
+     * If `regexps` is an empty array or `undefined`, then the function
```

Comment:
we will need to revisit those docs carefully, as it works differently now (again, by default, same subdomain strategy is used)

i will add a task to the roadmap, no need to deal with this here/now

Reply:
There's no strategy at all. It works with clicking elements, and there's no default `a` selector. So it works with all links produced by page after clicking the elements with provided selector. It's either then checks it vs patterns or enqueues it all.

So it's not just copied from enqeueLinks:

> If `pseudoUrls` is an empty array or `undefined`, then the function
> enqueues all the intercepted navigation requests produced by the page
> after clicking on elements matching the provided CSS selector.